lib/host.py

```diff
@@ -516,10 +516,12 @@ def disks(self):
         disks.sort()
         return disks
 
-    def disk_is_available(self, disk):
+    def disk_is_available(self, disk, disk_raw=False):
```

Comment:
I'm not sure this new param is explicit for everyone.
I prefer something like: `check_unformatted`, `check_no_fs` or `check_wiped`.

Reply:
I also find its naming unclear

Reply:
We can rename the parameter from `disk_raw` to `check_raw`, else for more elaborate use `type` option can be introduced. Where `type` can be `raw`, `LVM2_member`, `ext3`, `part` etc. and it can be found from `blkid` or suitable disk management command based on `type`.

e.g `disk_is_available(self, disk, type='raw')`

Reply:
I have added `raw_disk_is_available()` and kept the current function untouched.